packages/cli/templates/scaffolder-module.yaml

```diff
@@ -0,0 +1,14 @@
+description: A module exporting custom actions for @backstage/plugin-scaffolder-backend
+template: ./scaffolder-module
+targetPath: plugins
+backendModulePrefix: true
```

Comment:
I wonder if we could find a way to generalize this a bit - perhaps by adding support for variables interpolated from the template inputs? Might end up looking something like:

```yaml
variables:
  - suffix: scaffolder-backend-module-{{ moduleId }}
```